src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -203,6 +206,38 @@ private CommandLineSourceRepositoryProvider GetSourceRepositoryProvider()
             return new CommandLineSourceRepositoryProvider(SourceProvider);
         }
 
+        private DependencyBehavior TryGetDependencyBehavior(string behaviorStr)
+        {
+            DependencyBehavior dependencyBehavior;
+
+            if (!Enum.TryParse<DependencyBehavior>(behaviorStr, /* ignoreCase */ true, out dependencyBehavior))
```

Comment:
you need to follow this up with a check to Enum.IsDefined .

For more details, read remarks : https://msdn.microsoft.com/en-us/library/dd783499(v=vs.110).aspx